drupal/usr/local/share/drupal/drupal_functions.sh

```diff
@@ -90,20 +90,104 @@ do_drupal_install() {
   # Drop the database if we need to force the install every time.
   if [ "${FORCE_DATABASE_DROP}" == 'true' ]; then
     echo 'Dropping the Drupal DB if it exists'
-    as_code_owner "drush sql-drop -y -r ${WEB_DIRECTORY}"
+    as_code_owner "drush ${DRUPAL_DRUSH_ALIAS} sql-drop -y -r ${WEB_DIRECTORY}"
```

Comment:
Change to `drush sql-drop` to eliminate the risk of accidentally destroying data in the wrong environment.